code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -58,65 +58,91 @@
-				/obj/item/clothing/head/cone = 1,
-				/obj/item/weapon/coin/silver = 1,
-				/obj/item/weapon/coin/twoheaded = 1,
-				/obj/item/weapon/contraband/poster = 1,
-				/obj/item/weapon/crowbar = 1,
-				/obj/item/weapon/crowbar/red = 1,
-				/obj/item/weapon/extinguisher = 11,
+				/obj/item/stack/sheet/metal{amount = 20} = 10,
+				/obj/item/stack/sheet/mineral/plasma{layer = 2.9} = 10,
+				/obj/item/stack/sheet/rglass = 10,
+				/obj/item/weapon/book/manual/engineering_construction = 10,
+				/obj/item/weapon/book/manual/engineering_hacking = 10,
+				/obj/item/clothing/head/cone = 10,
+				/obj/item/weapon/coin/silver = 10,
+				/obj/item/weapon/coin/twoheaded = 10,
+				/obj/item/weapon/contraband/poster = 10,
+				/obj/item/weapon/crowbar = 10,
+				/obj/item/weapon/crowbar/red = 10,
+				/obj/item/weapon/extinguisher = 90,
 				//obj/item/weapon/gun/projectile/revolver/russian = 1, //disabled until lootdrop is a proper world proc.
-				/obj/item/weapon/hand_labeler = 1,
-				/obj/item/weapon/paper/crumpled = 1,
-				/obj/item/weapon/pen = 1,
-				/obj/item/weapon/plantspray/pests = 1,
-				/obj/item/weapon/stock_parts/cell = 3,
-				/obj/item/weapon/storage/belt/utility = 2,
-				/obj/item/weapon/storage/box = 2,
-				/obj/item/weapon/storage/box/cups = 1,
-				/obj/item/weapon/storage/box/donkpockets = 1,
-				/obj/item/weapon/storage/box/lights/mixed = 3,
-				/obj/item/weapon/storage/fancy/cigarettes/dromedaryco = 1,
-				/obj/item/weapon/storage/toolbox/mechanical = 1,
-				/obj/item/weapon/screwdriver = 3,
-				/obj/item/weapon/tank/emergency_oxygen = 2,
-				/obj/item/weapon/vending_refill/cola = 1,
-				/obj/item/weapon/weldingtool = 3,
-				/obj/item/weapon/wirecutters = 1,
-				/obj/item/weapon/wrench = 4,
-				/obj/item/weapon/relic = 3,
-				"" = 11
+				/obj/item/weapon/hand_labeler = 10,
+				/obj/item/weapon/paper/crumpled = 10,
+				/obj/item/weapon/pen = 10,
+				/obj/item/weapon/plantspray/pests = 10,
+				/obj/item/weapon/stock_parts/cell = 30,
+				/obj/item/weapon/storage/belt/utility = 20,
+				/obj/item/weapon/storage/box = 20,
+				/obj/item/weapon/storage/box/cups = 10,
+				/obj/item/weapon/storage/box/donkpockets = 10,
+				/obj/item/weapon/storage/box/lights/mixed = 30,
+				/obj/item/weapon/storage/fancy/cigarettes/dromedaryco = 10,
+				/obj/item/weapon/storage/toolbox/mechanical = 10,
+				/obj/item/weapon/screwdriver = 30,
+				/obj/item/weapon/tank/emergency_oxygen = 20,
+				/obj/item/weapon/tank/emergency_oxygen/engi = 10,
+				/obj/item/weapon/vending_refill/cola = 10,
+				/obj/item/weapon/weldingtool = 30,
+				/obj/item/weapon/wirecutters = 10,
+				/obj/item/weapon/wrench = 40,
+				/obj/item/weapon/relic = 30,
+				/obj/item/clothing/shoes/brown = 30,
+				////////////////CONTRABAND STUFF//////////////////
+				/obj/item/weapon/grenade/clown_grenade = 5,
+				/obj/item/seeds/ambrosiavulgarisseed/cruciatus = 5,
+				/obj/item/clothing/under/contortionist = 5,
```

Comment:
I don't think it's a good idea to give people something that lets them ventcrawl around.